cirq-core/cirq/sim/clifford/act_on_stabilizer_ch_form_args_test.py

```diff
@@ -117,3 +117,12 @@ def test_copy():
     assert args.prng is args1.prng
     assert args.log_of_measurement_results is not args1.log_of_measurement_results
     assert args.log_of_measurement_results == args1.log_of_measurement_results
+
+
+def test_deprecated():
```

Comment:
Nit: shouldn't this be in `act_on_stabilizer_args_test.py` instead?

Reply:
Dang I can't sneak anything by you. :) There *is* no `act_on_stabilizer_args_test.py`. I'll create one and add some other tests to that too.